packages/gemini-cli/build.sh

```
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -euo pipefail
+
+npm install -g --prefix=$OUTPUT_DIR/usr @google/gemini-cli@$MINIMAL_ARG_VERSION
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)